tests/integration/admin/test_describe_consumer_groups_compatability.py

```diff
@@ -0,0 +1,121 @@
+            "old": ["test-client3", "test-client4"],
+        }
+
+        consumers = [
+            create_consumers(kafka_cluster, topic, group_ids["new1"], client_ids["new"][0], "consumer"),
+            create_consumers(kafka_cluster, topic, group_ids["new2"], client_ids["new"][1], "consumer"),
+            create_consumers(kafka_cluster, topic, group_ids["old1"], client_ids["old"][0], "classic"),
+            create_consumers(kafka_cluster, topic, group_ids["old2"], client_ids["old"][1], "classic"),
+        ]
+        return group_ids, client_ids, consumers
+
+    def verify_consumer_group_results(fs, expected_group_ids, expected_type, expected_clients):
+        """Verify the results of consumer group descriptions."""
+        for group_id, f in fs.items():
+            result = f.result()
+            assert result.group_id in expected_group_ids
+            assert result.is_simple_consumer_group is False
+            assert result.state == ConsumerGroupState.STABLE
+            assert result.type == expected_type
+            assert len(result.members) == 1
+            for member in result.members:
+                assert member.client_id in expected_clients
+                assert member.assignment.topic_partitions == partition
+
+    # Create consumer groups
+    group_ids, client_ids, consumers = create_consumer_groups()
+    partition = [TopicPartition(topic, 0)]
+
+    # Describe and verify new group protocol consumer groups
+    fs_new = admin_client.describe_consumer_groups([group_ids["new1"], group_ids["new2"]])
+    verify_consumer_group_results(fs_new, [group_ids["new1"], group_ids["new2"]],
+                                  ConsumerGroupType.CONSUMER, client_ids["new"])
+
+    # Describe and verify old group protocol consumer groups
+    fs_old = admin_client.describe_consumer_groups([group_ids["old1"], group_ids["old2"]])
+    verify_consumer_group_results(fs_old, [group_ids["old1"], group_ids["old2"]],
+                                  ConsumerGroupType.CLASSIC, client_ids["old"])
+
+    # Describe and verify all consumer groups
+    fs_all = admin_client.describe_consumer_groups(list(group_ids.values()))
+    for group_id, f in fs_all.items():
+        result = f.result()
+        assert result.group_id in group_ids.values()
+        assert result.is_simple_consumer_group is False
+        assert result.state == ConsumerGroupState.STABLE
+        if result.group_id in [group_ids["new1"], group_ids["new2"]]:
+            assert result.type == ConsumerGroupType.CONSUMER
+            assert result.members[0].client_id in client_ids["new"]
+        else:
+            assert result.type == ConsumerGroupType.CLASSIC
+            assert result.members[0].client_id in client_ids["old"]
+        assert result.members[0].assignment.topic_partitions == partition
+
+    # Close all consumers
+    for consumer in consumers:
+        consumer.close()
+
+
+def test_describe_consumer_groups_compatability(kafka_cluster):
```

Comment:
```suggestion
def test_describe_consumer_groups_compatibility(kafka_cluster):
```

Reply:
We should change the test test_describe_operations.py instead of adding this one. There's a test that is disabled with KIP-848 there.
Here it seems the `verify_describe_consumer_groups` test isn't run as it doesn't start with `test_` and isn't called by `test_describe_consumer_groups_compatibility`